actors/builtin/miner/policy.go

```diff
@@ -71,47 +76,52 @@ var MaxSealDuration = map[abi.RegisteredSealProof]abi.ChainEpoch{
 
-// Minimum period before a deadline's challenge window opens that a fault must be declared for that deadline.
-// This lookback must not be less than WPoStChallengeLookback lest a malicious miner be able to selectively declare
-// faults after learning the challenge value.
+// Filecoin Parameter: Minimum period between fault declaration and the next deadline opening.
+// If the number of epochs between fault declaration and deadline's challenge window opening is lower than FaultDeclarationCutoff,
+// the fault declaration is considered invalid for that deadline.
+// Motivation: This guarantees that a miner is not likely to successfully fork the chain and declare a fault after seeing the challenges.
 const FaultDeclarationCutoff = WPoStChallengeLookback + 50
 
-// The maximum age of a fault before the sector is terminated.
+// Filecoin Parameter: The maximum age of a fault before the sector is terminated.
+// Motivation: This guarantees to clients that a Filecoin miner cannot lose the file for longer than 14 days.
 const FaultMaxAge = WPoStProvingPeriod * 14
 
-// Staging period for a miner worker key change.
-// Finality is a harsh delay for a miner who has lost their worker key, as the miner will miss Window PoSts until
+// Filecoin Parameter: Staging period for a miner worker key change.
+// Future improvement: Finality is a harsh delay for a miner who has lost their worker key, as the miner will miss Window PoSts until
 // it can be changed. It's the only safe value, though. We may implement a mitigation mechanism such as a second
 // key or allowing the owner account to submit PoSts while a key change is pending.
+// Motivation: This guarantees that a miner cannot choose a more favorable worker key that wins leader elections.
 const WorkerKeyChangeDelay = ChainFinality
 
-// Maximum number of epochs past the current epoch a sector may be set to expire.
+// Filecoin Parameter: Maximum number of epochs past the current epoch a sector may be set to expire.
 // The actual maximum extension will be the minimum of CurrEpoch + MaximumSectorExpirationExtension
 // and sector.ActivationEpoch+sealProof.SectorMaximumLifetime()
 const MaxSectorExpirationExtension = builtin.EpochsInYear
 
-// Ratio of sector size to maximum deals per sector.
+// Filecoin Parameter: Ratio of sector size to maximum deals per sector.
 // The maximum number of deals is the sector size divided by this number (2^27)
 // which limits 32GiB sectors to 256 deals and 64GiB sectors to 512
 const DealLimitDenominator = 134217728
 
+// Filecoin Parameter: Quality multiplier for CC sectors
 var QualityBaseMultiplier = big.NewInt(10)
+// Filecoin Parameter: Quality multiplier for a standard deal in a sector
 var DealWeightMultiplier = big.NewInt(10)
+// Filecoin Parameter: Quality multiplier for verified deals
 var VerifiedDealWeightMultiplier = big.NewInt(100)
-
+// Filecoin Parameter: Precision for Quality Adjusted power
 const SectorQualityPrecision = 20
```

Comment:
**Clarification Request:** Why is this set to 20?

Reply:
SectorQuality of a sector can be a fraction and we think a 2^20 fixed precision floating point is a good enough precision. cc @davidad